server/services/visitSessionsService.ts

```diff
@@ -174,11 +174,11 @@ export default class VisitSessionsService {
   async updateVisit({
     username,
     visitData,
-    visitStatus = 'RESERVED',
+    visitStatus,
```

Comment:
Stop `updateVisit` defaulting to `RESERVED` to avoid inadvertently 'downgrading' a booking - that could then get housekept.